### DISCUSSION TITLE
Planing and Milestones

### DISCUSSION THREAD

**Discussion by saeb-panahifar** [ChapterII/ngx-agora-sdk-ng#23] [General]
I created this discussion for planing. We could talk about sprint planning here.

**Comment by saeb-panahifar** [comment]
**Q** : means **Quarter** 
**M** : means **Milestone** 

Each year have 4 quarter.
Each milestone is 2 weeks.

**Comment by maryam0066** [comment]
OK Great

**Comment by amirber** [comment]
tnx
